### DISCUSSION TITLE
Importing Custom Fonts

### DISCUSSION THREAD

**Discussion by alibbosca** [jakewies/hugo-theme-codex#158] [Q&A] [answered]
Hi All,

I was working with the theme and was curious as to how custom fonts could be imported. I was able to change the fonts using the supported fonts but was unable to find a way to import my own fonts. Would appreciate any help.

Thanks!

**Comment by lzlrd** [comment, marked as answer]
Here's an example on how to do so, https://github.com/lazerl0rd/hugo-theme-codex/commit/879384425fa81057687a2e7b0443250d4bb7df83.
